client/deposits/list/index.tsx

```diff
@@ -209,50 +223,51 @@ export const DepositsList = (): JSX.Element => {
 
 	const downloadable = !! rows.length;
 
-	const onDownload = async () => {
-		setIsDownloading( true );
-		const downloadType = totalRows > rows.length ? 'endpoint' : 'browser';
+	const endpointExport = async ( language: string ) => {
+		// We destructure page and path to get the right params.
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we need to suppress the linter warning here?

Reply:
It's a remnant from the previous method. Removed it.